sentry.go

```diff
@@ -79,14 +85,29 @@ func (so *SentryOutput) getClient(dsn string) (client *raven.Client, err error)
 
 func (so *SentryOutput) Run(or pipeline.OutputRunner, h pipeline.PluginHelper) (err error) {
 	var (
-		e      error
-		pack   *pipeline.PipelinePack
-		client *raven.Client
+		e        error
+		pack     *pipeline.PipelinePack
+		client   *raven.Client
+		contents []byte
 	)
 
+	encoder := or.Encoder()
+	if encoder == nil {
+		payloadEncoder := new(plugins.PayloadEncoder)
+		payloadEncoder.Init(new(plugins.PayloadEncoderConfig))
+		encoder = payloadEncoder
+	}
+
 	sentryMsg := &SentryMsg{}
 
 	for pack = range or.InChan() {
+		contents, e = encoder.Encode(pack)
```

Comment:
The recommended pattern is to call or.Encode(), rather than to grab the encoder instance.